src/component/mxgraph/MxGraphConfigurator.ts

```diff
@@ -55,5 +56,15 @@ export default class MxGraphConfigurator {
     // Disable folding for container mxCell (pool, lane, sub process, call activity) because we don't need it.
     // This also prevents requesting unavailable images (see #185) as we don't override mxGraph folding default images.
     this.graph.foldingEnabled = false;
+
+    // Pan configuration
+    if (options?.mouseNavigationSupport) {
+      this.graph.panningHandler.useLeftButtonForPanning = true;
+      this.graph.panningHandler.ignoreCell = true; // ok here as we cannot select cells
+      this.graph.setPanning(true);
+    } else {
+      this.graph.setPanning(false);
+      this.graph.panningHandler.setPinchEnabled(false); // ensure gesture support is disabled (pan and zoom)
```

Comment:
:information_source: I have not test that, but for sure, prior this change, gesture was on (for pan and zoom). This doesn't work well when using a floating viewport size.

Reply:
I will test the pinch for zoom